backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -155,6 +155,11 @@ def spree_dom_id(record)
         dom_id(record, 'spree')
       end
 
+      def admin_layout(layout = nil)
+        @admin_layout = layout if layout
+        @admin_layout
+      end
+
```

Comment:
Okay, I think this is confusing enough that some documentation is def going to be required for people to figure it out. I still suggest that in the Rails context using the word 'layout' as something you can set, to a value that has nothing to do with the Rails concept of 'layout' (which is a template, or the name of a template)... might be worth considering if that's a good idea, in an already somewhat confusing setup.